package.json

```diff
@@ -17,10 +17,6 @@
     "coverage": "aegir coverage",
     "coverage-publish": "aegir coverage publish"
   },
-  "pre-commit": [
```

Comment:
This should not be removed, but changed to pre-push. I'd also like to see a better commit message :)

Reply:
@diasdavid why _was_ the `pre-commit` check removed?

@vmx is `pre-push` the new `pre-commit`? I don't think the [`pre-commit`](https://github.com/observing/pre-commit/blob/master/README.md) module we use supports `pre-push`.

Reply:
@olizilla `pre-push` is indeed the new `pre-commit`. You don't need to add the package, it's part of AEgir.